### PR TITLE
false positive.

### PR DESCRIPTION
**Pull request by yanivkalfa**
Do you really want to notify that its built even while there's an error ?

### CODE REVIEW

**Comment by ctrlplusb** [comment]
Yeah, probably not what I am wanting to do.  Thanks for spotting that! :)

I've made a small comment on your code. Happy to merge thereafter.

File: tools/development/server.js [hunk 1]
```diff
@@ -251,13 +251,13 @@ class HotServers {
         });
         console.log(stats.toString());
         return;
```
Comment (ctrlplusb) [inline tools/development/server.js]: Doesn't this return statement prevent any of the following lines from executing in the case of an error?

**Comment by yanivkalfa** [comment]
ah wow, i am blind, did not see this return. sorry.

**Comment by ctrlplusb** [comment]
no problem, this happens sometimes!  thanks for caring enough to PR though! :)

**Pull request closed without merging**